### ISSUE TITLE
vscode://file/ URI not working from chrome

### ISSUE THREAD

**Issue by rualark** [microsoft/vscode#94375]
- VSCode Version: 1.43.2
- OS Version: Ubuntu 18.04

Steps to Reproduce:

1. Click on vscode://file/xxx link in chrome

Does this issue occur when all extensions are disabled?: Yes

**Comment by joaomoreno** [comment]
Are you using the snap package?

**Comment by rualark** [comment]
Yes, I am using vscode snap installed from Ubuntu Software

**Comment by joaomoreno** [comment]
Exactly, this is a known snap issue. The snap team has some thoughts on it. cc @flexiondotorg @sergiusens

**Comment by rualark** [comment]
Is there a workaround? Should I install vscode or chromium not from snap?

**Comment by joaomoreno** [comment]
Not really sure. This is still new grounds. If you can figure out a workaround let us know, so we get closer to a solution!
